modules/installation-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+//* installing/installing-vsphere-installer-provisioned-customizations.adoc [IPI]
+//* installing/installing-vsphere-installer-provisioned-network-customizations.adoc [IPI]
+//* installing/installing-vsphere.adoc [UPI]
+//* installing/installing-vsphere-network-customizations.adoc [UPI]
+//* installing/installing-restricted-networks-installer-provisioned-vsphere.adoc [IPI]
+//* installing/installing-restricted-networks-vsphere.adoc [IPI]
+
+:_content-type: CONCEPT
+[id="installation-vsphere-regions-zones_{context}"]
+= VMware vSphere region and zone enablement
+
+You can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. Each datacenter can run multiple clusters. This configuration reduces the risk of a hardware failure or network outage that can cause your cluster to fail.
+
+:FeatureName: VMware vSphere region and zone enablement
+include::snippets/technology-preview.adoc[]
+
+The default installation configuration deploys a cluster to a single vSphere datacenter. If you want to deploy a cluster to multiple vSphere datacenters, you must create an installation configuration file that enables the region and zone feature.
+
+[IMPORTANT]
+====
+The VMware vSphere region and zone enablement feature requires the vSphere Container Storage Interface (CSI) driver as the default storage driver in the cluster. As a result, the feature only available on a newly installed cluster.
+
+A cluster that was upgraded from a previous release defaults to using the in-tree vSphere driver, so you must enable CSI automatic migration for the cluster. You can then configure multiple regions and zones for the upgraded cluster.
+====
+
+The default `install-config.yaml` file includes `vcenters` and `failureDomains` fields, where you can specify multiple vSphere datacenters and clusters for your {product-title} cluster. You can leave these fields blank if you want to install an {product-title} cluster in a vSphere environment that consists of single datacenter.
+
+The following list describes terms associated with defining zones and regions for your cluster:
+
+* Failure domain: Establishes the relationships between a region and zone. You define a failure domain by using vCenter objects, such as a `datastore` object. A failure domain defines the vCenter location for {product-title} cluster nodes.
```

Comment:
If you want these in bold, try `Failure domain:: `. Checkout [this procedure](https://docs.openshift.com/container-platform/4.12/post_installation_configuration/machine-configuration-tasks.html#checking-mco-status_post-install-machine-configuration-tasks), step 1 following "where:"

Reply:
Good suggestion. I tried this during my newbie creativity moments, but it conflicted with the style applied on other installation doc sets, so I reverted the change. The list as is will suffice and has passed approvals on 4.13. Thanks!